airmozilla/main/fields.py

```diff
@@ -1,7 +1,7 @@
 from django.db import models
 from django.forms.fields import Field
 from django.forms.util import ValidationError
-from django.utils import simplejson as json
+import json
```

Comment:
@aarshvig because this is now your second patch, it's time to start getting more and more perfect. 

So, what you need to do is to move this line up to the top of the file. There's a stylistic pattern. You'll find this in almost every other file. It goes like this:
```
import some
import standard
import python
import library
[EMPTY LINE]
import some
import third_party
import python
import library
[EMPTY LINE]
import things
import from
import django
[EMPTY LINE]
import third
import party
import django
import related
import libraries
[EMPTY LINE]
import airmozilla
import specific
import modules
import and
import libraries
```

Look at [main/views.py](https://github.com/mozilla/airmozilla/blob/master/airmozilla/main/views.py#L1-L56) for example.

Does that make sense?